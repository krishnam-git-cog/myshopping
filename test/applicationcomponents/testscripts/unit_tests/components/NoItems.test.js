import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import NoItems from '../../../../../src/components/noitems/NoItems';

describe('NoItems test', () => {
    it("Component - NoItems should match snapshot",  () => {
       
          let shallow = createShallow();
          const component = shallow(<NoItems />);
          expect(component).toMatchSnapshot();
      });
});