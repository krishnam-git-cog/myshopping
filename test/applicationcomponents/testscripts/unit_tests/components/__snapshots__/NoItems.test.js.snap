// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoItems test Component - NoItems should match snapshot 1`] = `ShallowWrapper {}`;
